walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -11,13 +11,64 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
 package com.google.sps;
 
-import java.util.Collection;
-
-public final class FindMeetingQuery {
+import java.util.*;
+ 
+public final class FindMeetingQuery { 
+  private ArrayList<TimeRange> viableTimeSlots = new ArrayList<TimeRange>(); 
```

Comment:
It doesn't seem like you need a global variable. You didn't clean up the variable which meant if you call Query() twice in a roll, you will have the wrong answer.
Instead of having the global variable, just make generatePotentialTimes return an ArrayList<TimeRange>.

Reply:
Oh wow, I suppose you're right, none of the tests did query twice in a row so I never realized this.